docs/developer.rst

```diff
@@ -88,23 +85,20 @@ for more details. Install the following packages::
   pacman -S patch realpath
 
 
-Once everything is installed. Open PowerShell, make sure you've added MSYS2
-path to current session. Ensure ``bazel``, ``patch`` and ``realpath`` are
-accessible. Activate conda environment. Folllowing command build with cuda
-enabled with PDB generated, adjust it to whatever suitable for you.::
+Once everything is installed. Open PowerShell, and make sure MSYS2 is in the
+path of the current session. Ensure ``bazel``, ``patch`` and ``realpath`` are
+accessible. Activate the conda environment. The folllowing command build with
```

Comment:
Nit:
"The following command builds with CUDA enabled; adjust it to whatever is suitable for you::"
(Typo in "following"; "builds" needs an "s", and we need a break before "adjust")